ImageFeed/Profile/ProfileService/Profile.swift

```diff
@@ -8,6 +8,7 @@
 import Foundation
 
 struct Profile: Decodable {
-    let username, firstName, lastName: String
+    let username, firstName: String
```

Comment:
Перечисление переменных одного тип в swift хоть и доступно, однако почти не встречается на практике. Ориентируйся в пользу полного описания:
```suggestion
    let username
    let firstName: String
```